dvc/remote/ssh/connection.py

```diff
@@ -240,8 +241,11 @@ def execute(self, cmd):
             import select
 
             got_chunk = False
-            readq, _, _ = select.select([stdout.channel], [], [], self.timeout)
-            for c in readq:
+            poll = select.poll()
+            poll.register(stdout.channel, POLLIN)
```

Comment:
Looks like this is not supported for windows and we'll have to use select() on it.